weave-js/src/common/util/links.tsx

```diff
@@ -94,19 +94,16 @@ export const TargetBlank: FCWithRef<
   React.AnchorHTMLAttributes<HTMLAnchorElement>,
   HTMLAnchorElement
 > = React.memo(
-  React.forwardRef(({children, ...passthroughProps}, ref) => {
+  React.forwardRef(({children, href, ...passthroughProps}, ref) => {
     // Enforce an absolute prefixed url for blank targets
-    if (
-      passthroughProps.href != null &&
-      !(passthroughProps.href.indexOf('://') > 0) // regex for http, file, s3, gs, etc
-    ) {
-      passthroughProps.href = getConfig().urlPrefixed(passthroughProps.href);
-    }
+    const parsedUrl = getAbsolutePrefixedUrl(href);
+
     return (
       // eslint-disable-next-line wandb/no-a-tags
       <A
         target="_blank"
         rel="noopener noreferrer"
+        href={parsedUrl}
```

Comment:
previously, if the href param wasn't present, it wouldn't get added. This will always add the href property (although in some cases it will be undefined) - does that affect anything? I don't think so, but worth checking..

Reply:
checked that having href=undefined will not just do nothing, so this change should be okay